docs/team/weilin1202.md

```diff
@@ -10,36 +10,39 @@ It provides a centralised platform for them to manage their work, contacts, and
 
 Given below are my contributions to the project.
 
-* **New Feature**: to be added soon
-  * What it does: to be added soon
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: *{to be added soon}*
-
-* **New Feature**: to be added soon
-  * What it does: to be added soon
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: *{to be added soon}*
+* **New Feature**: Added an event tagging feature to the `add` and `edit` command 
```

Comment:
I think you can include the link to the PR here!

Reply:
Added!